ts/components/conversation/SessionConversation.tsx

```diff
@@ -55,9 +55,10 @@ import { MessageDetail } from './message/message-item/MessageDetail';
 
 import { HTMLDirection } from '../../util/i18n';
 import { SessionSpinner } from '../basic/SessionSpinner';
+import { deleteMessagesByIdForEveryone } from '../../interactions/conversations/unsendingInteractions';
 
 const DEFAULT_JPEG_QUALITY = 0.85;
-
+// tslint:disable: jsx-curly-spacing
```

Comment:
is this needed at all otherwise lgtm

Reply:
conflicts need to be resolved with unstable

Reply:
Rebased and resolved comment